File: 2020/02/1.jl
function is_valid_line(line)
   m = match(r"(\d+)-(\d+) (.): (.+)", line)
   min = parse(Int32, m[1])
   max = parse(Int32, m[2])
   c = m[3]
   pass = m[4]

   return min <= sum(split(pass, "") .== c) <= max
end

open("input") do f
    map(is_valid_line, readlines(f)) |> sum |> println
end

File: 2015/02/1.py
s = 0
for p in open("input"):
    l, w, h = map(int, p.split('x'))
    s1, s2, s3 = l*w, w*h, h*l
    s += 2*s1 + 2*s2 + 2*s3 + min(s1, s2, s3)
print(s)
